app/controllers/authenticate_controller.rb

```diff
@@ -127,6 +127,12 @@ def authenticate_jwt
     )
     render_authn_token(authn_token)
   rescue => e
+    # At this point authenticator_input.username is always empty (e.g. cucumber:user:USERNAME_MISSING)
+    log_audit_failure(
```

Comment:
Is not there original Audit event for JWT authentication ? Like for all other types of authentication ?